tests/functional/test_location_change.py

```diff
@@ -0,0 +1,38 @@
+import pytest
+import os
+from dbt.tests.util import run_dbt_and_capture
+
+_MODEL_SQL = """
+select 1 as id
+"""
+
+_INVALID_LOCATION = os.getenv('DBT_TEST_BIGQUERY_BAD_LOCATION', 'northamerica-northeast1')
+
+class BaseBigQueryLocation:
+
+    @pytest.fixture(scope="class")
+    def models(self):
+        return {
+            "model.sql": _MODEL_SQL,
+        }
+
+
+class TestBigqueryValidLocation(BaseBigQueryLocation):
+
+    def test_bigquery_valid_location(self, project):
+        _, stdout = run_dbt_and_capture()
+        assert "Completed successfully" in stdout
```

Comment:
Naive question: Why not just a `dbt run`? What makes this special that it requires a message string check?

Reply:
was playing around with various ways to grab adapter_response wanted to at least be able to check stdout from call to see of model passes or fails as expected (believe second model should fail functionally as we are changing location mid test). a `dbt_run` could be used will tinker a little more.

Reply:
no problem. Thanks for the context